docker-compose.yml

```diff
@@ -14,10 +14,16 @@ services:
       MYSQL_ROOT_PASSWORD: root
   openemr:
     restart: always
-    image: openemr/openemr:5.0.3
+    image: quay.io/openemr/openemr:latest
+    build:
+      context: images/openemr
+      args:
+        OPENEMR_REPO_URL: https://github.com/openemr/openemr.git
+        OPENEMR_BRANCH: master
+        ENABLE_SSL: "false"
     ports:
-    - 80:80
-    - 443:443
+    - 8080:8080
```

Comment:
The configs "listen on 8080". I do need to update the SSL port to 8443. I'll do a quick edit this morning